firefox-ios/WidgetKit/DownloadManager/DownloadLiveActivity.swift

```diff
@@ -53,12 +55,60 @@ struct DownloadLiveActivityAttributes: ActivityAttributes {
 @available(iOS 16.2, *)
 struct DownloadLiveActivity: Widget {
     var body: some WidgetConfiguration {
-        ActivityConfiguration(for: DownloadLiveActivityAttributes.self) { _ in
+        ActivityConfiguration(for: DownloadLiveActivityAttributes.self) { LiveDownload in
             // Using Rectangle instead of EmptyView because the hitbox
             // of the empty view is too small (likely non existent),
             // meaning we'd never be redirected to the downloads panel
-            Rectangle()
-                .widgetURL(URL(string: URL.mozInternalScheme + "://deep-link?url=/homepanel/downloads"))
+            ZStack {
+                Rectangle()
+                    .widgetURL(URL(string: URL.mozInternalScheme + "://deep-link?url=/homepanel/downloads"))
+                    .foregroundStyle(LinearGradient(
+                        gradient: Gradient(colors: [Color("searchButtonColorTwo"), Color("searchButtonColorOne")]),
+                        startPoint: .topLeading,
+                        endPoint: .bottomTrailing
+                    ))
+                HStack(spacing: 16) {
+                    ZStack {
+                        Image("faviconFox")
+                            .resizable()
+                            .scaledToFit()
+                            .frame(width: 40, height: 40)
+                    }
+                    VStack(alignment: .leading, spacing: 4) {
+                        Text("Downloading \(LiveDownload.state.downloads[0].fileName)")
+                            .font(.system(size: 17, weight: .bold))
+                            .foregroundColor(Color("widgetLabelColors"))
+                        Text("\(String(format: "%.1f", Double(LiveDownload.state.totalBytesDownloaded) / 1000000)) MB of \(String(format: "%.1f", Double(LiveDownload.state.totalBytesExpected) / 1000000)) MB")
```

Comment:
For this line, when determining the download progress string, to calculate the size lets do something similar to what the download toast does: 
https://github.com/mozilla-mobile/firefox-ios/blob/8ea7f2fb6bf6b86b8eddb218fb9c2ad383ca882c/firefox-ios/Client/Frontend/Browser/DownloadToast.swift#L97